yasa/hypno.py

```diff
@@ -83,6 +83,15 @@ class Hypnogram:
     >>> from yasa import Hypnogram
     >>> values = ["W", "W", "W", "S", "S", "S", "S", "S", "W", "S", "S", "S"]
     >>> hyp = Hypnogram(values, n_stages=2)
+    >>> hyp
+    <Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>
```

Comment:
Maybe an if statement to add scorer if present:

- if None: `<Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>`
- if YASA: `<Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages, scored by YASA>`

This one I'm not sure about so do whatever you think is best.

Reply:
Good idea, done!